src/category_theory/sites/types.lean

```diff
@@ -54,6 +54,11 @@ theorem is_sheaf_yoneda' {α : Type u} : is_sheaf types_grothendieck_topology (y
   by { convert this, exact rfl },
 λ f hf, funext $ λ y, by convert congr_fun (hf _ (hs y)) punit.star⟩
 
+/-- The yoneda functor that sends a type to a sheaf over the category of types -/
+def yoneda' : Type u ⥤ Sheaf types_grothendieck_topology :=
```

Comment:
Can you add @[simps] to this, and show that composing this with the forgetful functor Sheaf => Presheaf is yoneda?

Reply:
Done.